packages/components/src/standalone-comparison-grid/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button } from '@wordpress/components';
+import './style.scss';
+import { ReactNode } from 'react';
+
+interface StandAloneComparisonGridColumn {
+	title: string;
+	features: ReactNode[];
+	introCopy?: string;
+	actionCopy: string;
+	action: () => void;
+}
+
+export interface StandAloneComparisonGridProps {
```

Comment:
```suggestion
interface Props {
```
We don't need to export this interface, if we need to use the props we can just use the react ComponentProps utils to get.


paYKcK-45y-p2